doc/modules/ROOT/pages/usage/server.adoc

```diff
@@ -63,6 +91,7 @@ interface (this output was simply generated with `--help`):
 -b/--bind ADDR              Bind address, by default "127.0.0.1".
 -h/--host ADDR              Host address to connect to when using --connect. Defaults to "127.0.0.1".
 -p/--port PORT              Start nREPL on PORT. Defaults to 0 (random port) if not specified.
+-s/--socket PATH            Start nREPL on filesystem socket at PATH.
```

Comment:
I wonder if `-u/--unix-socket` won't be better (clearer) names here.

Reply:
Just fine with me if you prefer.  Would you also like me to change `:socket` to `:unix-socket`?

Oh, and haven't thought about it carefully yet, but remembered that as another approach, some tools like `psql` (postgresql cli) just treat any `--host` that starts with a slash as a filesystem socket path.

Reply:
@rlbdv Hmm, the idea to recycle `--host` is not bad either. Without giving it much thought - it definitely makes sense to me and saves us from having to come up with a name for this CLI option. :D